packages/main/src/Input.js

```diff
@@ -508,6 +545,12 @@ class Input extends UI5Element {
 		}
 	}
 
+	_handleResize() {
+		if (this.hasValueStateMessage) {
+			this._getPopover().header[0].style.width = `${this.offsetWidth}px`;
```

Comment:
Same, this should be a metadata private property, which will invalidate the input and set the style in the template

Reply:
Done.